src/post/Write/Write.js

```diff
@@ -60,6 +65,8 @@ class Write extends React.Component {
     createPost: () => {},
     saveDraft: () => {},
     notify: () => {},
+    replace: () => {},
+    draftId: null,
   };
 
   constructor(props) {
```

Comment:
Should this state be also outside of constructor?

Reply:
I wouldn't change unrelated code for now.